react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,26 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(arg, ...argsep) {
+  if (arg) {
+    if (isArray(arg)) return Math.min(...arg);
+    return Math.min(arg, ...argsep);
+  }
+  return undefined;
 }
 
-export function copy() {
+export function copy(arg) {
+  if (isArray(arg)) {
+    return [...arg];
+  }
+  const { a, b } = arg;
```

Comment:
This is an error, the object {a, b} is always returned but this function must create a copy of the arguments.

For the test it works, because it passes the necessary properties, but this function does not send a copy
! [image] (https://user-images.githubusercontent.com/26824183/62159185-bb803d80-b2d6-11e9-94ab-916f636240f6.png)